source/loader/layers/sanitizer/ur_sanitizer_utils.cpp

```diff
@@ -72,6 +72,22 @@ ur_device_handle_t GetDevice(ur_queue_handle_t Queue) {
     return Device;
 }
 
+std::vector<ur_device_handle_t> GetDevices(ur_context_handle_t Context) {
+    std::vector<ur_device_handle_t> Devices{};
+    uint32_t DeviceNum;
```

Comment:
```suggestion
    uint32_t DeviceNum = 0;
```

Reply:
it would be better to check the value of DeviceNum after urDdiTable.Context.pfnGetInfo, and make an earlier return if its value is 0

Reply:
Or check "Result"